app/api/events.py

```diff
@@ -33,16 +35,36 @@ class Meta:
         self_view_many = 'v1.event_list'
         inflect = dasherize
 
+    @validates_schema
+    def validate_date(self, data):
+        if data['starts_at'] >= data['ends_at']:
+            raise UnprocessableEntity({'pointer': 'ends_at'}, "ends_at should be after starts_at")
```

Comment:
IMO `pointer` seems the best as it is regularly used in examples in http://jsonapi.org/examples/#error-objects